### PR TITLE
WIP: Merge pubsub from master into feature/issue169

### PR DESCRIPTION
**Pull request by brookisme**
WIP for #169.

### CODE REVIEW

**Comment by brookisme** [comment]
cc: @eightysteele

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/wri/gfw-api.svg?branch=feature%2Fv1)](https://travis-ci.org/wri/gfw-api) [![Code Health](https://landscape.io/github/wri/gfw-api/feature/v1/landscape.png)](https://landscape.io/github/wri/gfw-api/feature/v1)
 
-# API Overview
+# API Overview (Test Edit 169)
```
Comment (brookisme) [inline README.md]: This looks great dude hh :)

**Comment by eightysteele** [comment]
NOTE - We're using GAE SDK 1.9.15 for the feature.

**Comment by brookisme** [comment]
This is ready to be merged with feature/v1.  Creating a new branch to work on digest emails.

**Comment by brookisme** [comment]
New tasks for this issue:
 [ ] change gamma on landsat tiles
 [ ] update umd to use 2014 data
 [ ] tree cover loss analysis
 [ ] digest emails

File: gfw/polyline.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
+'''Provides utility functions for encoding and decoding linestrings using the
```
Comment (eightysteele) [inline gfw/polyline.py]: remove this file from PR

File: gfw/forma.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
+# Global Forest Watch API
```
Comment (eightysteele) [inline gfw/forma.py]: remove this, use/update gfw/forestchange/forma.py instead

File: gfw/api.py [hunk 4]
```diff
@@ -0,0 +1,354 @@
+# Global Forest Watch API
```
Comment (eightysteele) [inline gfw/api.py]: remove this file, instead route pubsub requests through gfw/forestchange/api.py and dispatch to a new gfw/forestchange/pubsub.py

File: gfw/stories.py [hunk 5]
```diff
@@ -0,0 +1,105 @@
+# Global Forest Watch API
```
Comment (eightysteele) [inline gfw/stories.py]: remove this for now, we will do separate PR for stories

File: gfw/wdpa.py [hunk 6]
```diff
@@ -0,0 +1,56 @@
+# Global Forest Watch API
```
Comment (eightysteele) [inline gfw/wdpa.py]: remove this file, use gfw/forestchange/wdpa.py instead

File: lib/arrow/__init__.py [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
```
Comment (eightysteele) [inline lib/arrow/__init__.py]: remove lib/arrow and add in a separate PR

File: lib/dateutil/__init__.py [hunk 8]
```diff
@@ -0,0 +1,9 @@
+"""
```
Comment (eightysteele) [inline lib/dateutil/__init__.py]: remove dateutil, add in separate PR

File: test/gfw/api_test.py [hunk 9]
```diff
@@ -0,0 +1,374 @@
+# Global Forest Watch API
```
Comment (eightysteele) [inline test/gfw/api_test.py]: merge this into the test/gfw/forestchange/pubsub_test.py or something
